bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -25,6 +27,8 @@
 class TalentPool(WatchChannel, Cog, name="Talentpool"):
     """Relays messages of helper candidates to a watch channel to observe them."""
 
+    talentpool_settings = RedisCache()
```

Comment:
Can you give this a description of what the keys and values are, similarly to the ones in modpings and voice_gate

Reply:
👍